.github/workflows/pr-requirements-check.yml

```
@@ -0,0 +1,40 @@
+      - dev
+
+jobs:
+  check-pr-requirements:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Set PR context
+      id: pr_context
+      run: |
```

actionlint (1.7.7)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)